vips/vip-106.ts

```diff
@@ -0,0 +1,204 @@
+import { ProposalType } from "../src/types";
```

Comment:
VIP sequence(name i.e. "vip-106") is same as in this PR https://github.com/VenusProtocol/vips/pull/5/files